_data/lifecycle-phases.yml

```
@@ -0,0 +1,159 @@
+phases:
+  - name: Proposal
+    description: |
+      Components under consideration for development through public discussion and a formal proposal.
+    learn_more: Visit the proposal process overview to learn more about how to contribute.
+    subphases:
+      - name: Discussion started
+        description: |
+          A component idea has been submitted to the discussion board.
+        starts_when: |
+          Anyone starts a discussion about a potential new component.
+        ends_when: |
+          The USWDS core team begins drafting a proposal for the component.
+        contribute_by:
+          - Opening a new component discussion
```

Need to add link once it is available.
```suggestion
          - Opening a [new component discussion]()
```